test/fail_compilation/fail18979.d

```diff
@@ -2,8 +2,9 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail18979.d(13): Deprecation: `imports.imp18979.Foo.__ctor(A)(A a)` is not visible from module `fail18979`
----
+fail_compilation/fail18979.d(14): Error: `imports.imp18979.Foo.__ctor(A)(A a)` is not visible from module `fail18979`
```

Comment:
Shouldn't it be an "undefined identifier" error ?
Also, in a perfect world, we shouldn't show "__ctor" to the user (or any symbol which is not defined in code), but that was already the case before.

Reply:
How about now?